src/drivers/gps/gps.cpp

```diff
@@ -688,6 +690,7 @@ GPS::run()
 
 			case GPS_DRIVER_MODE_ASHTECH:
 				_helper = new GPSDriverAshtech(&GPS::callback, this, &_report_gps_pos, _p_report_sat_info, heading_offset);
+				_report_gps_pos.heading_offset = heading_offset;
```

Comment:
Can you set this unconditionally for all drivers, so that it automatically works when we add heading support to another driver?

Reply:
Will do.